apps/store/extensions/app/social-reviews/modules/reviews-utils.js

```diff
@@ -43,37 +46,65 @@ var ReviewUtils = {};
         paging.limit = paging.limit ? paging.limit : REVIEW_PAGE_LIMIT;
         return paging;
     };
-    var processUserReviews = function(socialSvc, user, reviews) {
+    var processUserReviews = function (socialSvc, user, reviews, target) {
         var review;
-        var formattedUsername = formatUsername(user);
+        var myReview;
+        var socialSvc = getSocialSvc();
+        var username = formatUsername(user);
+        myReview = JSON.parse(String(socialSvc.getUserComment(username, target)));
+        if (!isEmpty(myReview)) {
+            myReview.actor.id = cleanUsername(myReview.actor.id);
+        }
+        else {
```

Comment:
Thank you, I have fixed this issue in following [commit](https://github.com/wso2/carbon-store/pull/643/commits/358a87f6fa0ffb067480daf1fe3e463bed87080e)